### ISSUE TITLE
Consider vertical navigation for guides

### ISSUE THREAD

**Issue by willguv** [localgovdrupal/localgov_guides#123]
Several LGD councils have switched the in-guide navigation list to vertical

eg

https://new.haringey.gov.uk/council-tax/council-tax-payments-refunds/pay-your-council-tax

https://www.essex.gov.uk/adult-social-care-and-health/blue-badge/apply-blue-badge/sending-documents

We need to:

- consider this as a default for new users
- provide an option for existing councils (some of their guides include many pages so list is long eg https://www.croydon.gov.uk/council-and-elections/privacy-and-data-protection/privacy-notices)
- update the guides doc to recommend 6-8 pages per guide

**Comment by willguv** [comment]
Option would be provided by block config, a non dev needs to be able to change this

**Comment by msayoung** [comment]
Noting this was raised in the session @Shaz46726 ran today. 

Someone noted that the current solution may be more tricky to navigate for people using screen magnification.

**Comment by willguv** [comment]
Chat in the Content Group just now

Mark suggested a checkbox to switch between horizontal and vertical layouts

Default should be vertical

**Comment by andybroomfield** [comment]
BHCC has also done this
https://www.brighton-hove.gov.uk/business-and-trade/apply-taxi-licence/apply-new-private-hire-vehicle-licence

**Comment by msayoung** [comment]
> Option would be provided by block config, a non dev needs to be able to change this

@willguv - this just came up in Merge Tuesday. What was the motivation to change it on a guide by guide basis? 

We were just discussing whether it would make more sense for it to be a sitewide settings, which would provide consistency across the site.

**Comment by finnlewis** [comment]
Discussing the pull request in Merge Tuesday https://github.com/localgovdrupal/localgov_guides/pull/182

Allowing each guide to have vertical or horizontal navigation introduces interesting challenges for deploying to existing sites. Maria then called out that maybe this should be a site-wide setting for consistency of UX, rather than being a content designer decision on each guide.

@willguv you mention above: 

> Option would be provided by block config, a non dev needs to be able to change this

Can we check, 

1. does a content designer really need to be able to set this on a guide by guide basis? 
2. why does a non dev need to be able to change this

Thanks @willguv !

**Comment by willguv** [comment]
Hi @msayoung @finnlewis sorry I wasn't there today

1. I don't think there would be any reason to change the setting on a guide by guide basis. It's definitely a site wide setting
2. Some councils have limited or no dev support so makes sense to provide UI for this

**Comment by markconroy** [comment]
@willguv PR available for this - https://github.com/localgovdrupal/localgov_base/pull/716

It has 2 things in it

1. Stacked header
2. Vertical navigation
